### PR TITLE
backend_pool: don't requeue item if not supported

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

File: src/core/backends/backend_pool.py [hunk 1]
```diff
@@ -26,18 +26,23 @@ def __init__(self, backend: Any, loop: asyncio.AbstractEventLoop):
         self.work_index = dict()
 
     def pick_tasks(self, queue: asyncio.Queue, n_tasks: int) -> List[Task]:
-        # TODO worker must support the items in the task in order to work on it
-        # need some additional item_list checks and some asyncio.Queue wrapper
-        # that makes sure task distribution happens correctly
+        # TODO some asyncio.Queue wrapper that makes sure task distribution happens correctly
```
Comment (maximumstock) [inline src/core/backends/backend_pool.py:29]:
```suggestion
```

Comment (maximumstock) [reply src/core/backends/backend_pool.py:29]: We don't need that anymore I think

**Comment by maximumstock** [comment]
LGTM 👍🏻  This was on my list but didn't go through with it after I noticed how borked the new rate-limits are. Thank you very much!

**Comment by 1480c1** [comment]
Yeah, this was motivated when I  noticed in --debug that it was looping on certain items and it didn't make sense to me for it to requeue the failing tasks when it was due to backend not supporting it

**Comment by maximumstock** [comment]
Thanks again for the initiative!

**Pull request merged**